README.md

```diff
@@ -36,13 +36,15 @@ concurrency:
 
 jobs:
   deploy_yard:
+    # the deploy environment (not to be confused with env)
     environment:
       name: github-pages
       url: ${{ steps.deployment.outputs.page_url }}
     runs-on: ubuntu-latest
     name: Build and deploy YARD
     steps:
       - uses: kachick/deploy-yard-to-pages@v1.2.0
+        id: deployment
```

Comment:
I think this might make confusion with actions/deploy-pages@v1

And I want to keep minimum for required config.

Reply:
without this change, the `enviroment` field above has an undefined URL and there is no `view deployment` button on PRs / on the deployments page

<img width="1459" alt="CleanShot 2023-01-04 at 15 28 19@2x" src="https://user-images.githubusercontent.com/60136/210576951-6afe0b27-ad9f-4e25-9f7c-1d14f358d265.png">

an alternative to this PR: I'd suggest removing the `url` from the `environment` field above.

Reply:
Oh... I never know the button exists in pages feature. 😮 🙇‍♂️